pkg/scheduler/framework/plugins/nodevolumelimits/csi.go

```diff
@@ -77,8 +80,82 @@ func (pl *CSILimits) Name() string {
 func (pl *CSILimits) EventsToRegister() []framework.ClusterEventWithHint {
 	return []framework.ClusterEventWithHint{
```

Comment:
I know it's not related to this PR, but what if PVC is not found and Pod is rejected in the scheduling cycle?
I guess the plugin should return UnschedulableAndUnresolvable in such case and we need to add PVC Added event to catch the moment when PVC for the Pod is created.

Reply:
gonna be fixed in https://github.com/kubernetes/kubernetes/pull/121952